### ISSUE TITLE
Users getting failed installs due to "Http Error Unauthorized - Unauthorized"

### ISSUE THREAD

**Issue by Mieekaserra** [wabbajack-tools/wabbajack#2491]
**Describe the bug**

As title, user states they installed DNGG fine, but when they closed and reopened wabbajack to install wildlander, this started happening.

**Additional Context**

A wabbajack reset or Loging out and logging back in fixes it.

**Wabbajack Version**

3.5.0.1

**Logs**

[message(1).txt](https://github.com/wabbajack-tools/wabbajack/files/14008144/message.1.txt)

**Comment by tr4wzified** [comment]
The only reason that would explain this happening is that the API key changed somehow. Maybe the user was messing around with that in their Nexus settings? Have you seen this more often, is it reproducible in any way?

**Comment by Mieekaserra** [comment]
I see it daily in the wabbajack help channel. Here was yesterday's

https://cdn.discordapp.com/attachments/666599178788536331/1203342171714097192/Wabbajack.current.log

**Comment by Mieekaserra** [comment]
I'm putting this back in because Luca misread it:P 

I see it roughtly every day in wabbajack help - here's a example from yesterday https://cdn.discordapp.com/attachments/666599178788536331/1203342171714097192/Wabbajack.current.log

**Comment by tr4wzified** [comment]
I don't think this is still an issue? If so it's a bit too vague to even start to investigate as it is now, the log posted is no longer available
